flogger/flogger-system-backend/src/main/java/com/google/common/flogger/backend/system/AbstractBackend.java

```diff
@@ -38,7 +38,8 @@ public abstract class AbstractBackend extends LoggerBackend {
 
   // Internal constructor used by legacy callers - should be updated to just pass in the logging
   // class name. This needs work to handle anonymous loggers however (if that's ever supported).
-  AbstractBackend(Logger logger) {
+  // Should be removed or become `internal` when migrated to Kotlin.
```

Comment:
To distinguish existing comments, and our action points, please use a `todo` template, along with the reference to the corresponding GitHub issue.

Such things should be visible.